modules/patterns/pages/github-merge-queues.adoc

```diff
@@ -0,0 +1,64 @@
+= GitHub Merge Queues
+
+This document contains instructions for how to configure configure GitHub link:https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue[merge queues] with {ProductName}.
+
+A merge queue is a GitHub feature that provides the same benefits as the **Require branches to be up to date before merging** branch protection, but does not require a pull request author to update their pull request branch and wait for status checks to finish before trying to merge.
+
+By leveraging merge queues, you can be more sure that competing changes landing on your main branch do not have conflicting changes that fail CI, but without the hassle of requiring you to manually rebase every pull request when another one merges.
+
+NOTE: GitHub merge queues are similar to both link:https://docs.prow.k8s.io/docs/components/core/tide/[tide] pools from the link:https://docs.prow.k8s.io/docs/[prow] system and link:https://docs.gitlab.com/ci/pipelines/merge_trains/[merge trains] from Gitlab.
+
+== Prerequisites
+
+You already have a Component onboarded to {ProductName} with working pipelines.
+
+== Insight
+
+When merge queues are enabled, GitHub will push your commits to a link:https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#triggering-merge-group-checks-with-third-party-ci-providers[special branch] (`gh-readonly-queue/{base_branch}`) before a change is merged to `{base_branch}`. We will configure Konflux to trigger pipelines in response to pushes to that branch.
```

Comment:
What we want to do here specifically is ensure that the artifacts can build? Are there any ITS contexts that are specifically relevant for merge queues?

Reply:
Hm, I don't know. I just leave it kind of open to the user's decision in the "Require status checks to pass before merging" bullet below. I also don't think any of the ITS contexts are *especially* relevant to merge queues, are they?

Reply:
Sounds good.

Considering the contexts, I don't think so. It would probably have to be a context that could also filter on the branch.